test/src/toga_test/app.py

```diff
@@ -0,0 +1,12 @@
+import toga
+
+
+class TogaTest(toga.App):
+    def startup(self):
+        self.main_window = toga.MainWindow(title=self.formal_name)
+        self.main_window.content = toga.Box()
+        self.main_window.show()
+
+
+def main():
+    return TogaTest(app_name="toga_test")
```

Comment:
This is odd... why is app_name needed here? It should be loading this from the `dist-info`...

Reply:
Oh - I get it now - it's because the entry point app is `tests.toga_test`, which doesn't have a corresponding dist-info; passing `app_name` is enough to point it at the right dist-info file.